apps/example/src/app/api/v2/route-with-external-dep/route.ts

```diff
@@ -0,0 +1,25 @@
+import { TypedNextResponse, route, routeOperation } from 'next-rest-framework';
+import { z } from 'zod';
+import { JSDOM } from 'jsdom';
```

Comment:
An example using `jsdom` that is now marked as `external` in our custom `openapi-generate.ts` file.